build.cake

```diff
@@ -16,8 +16,13 @@ Task("Clean")
         CleanDirectory("./build/obj");
         CleanDirectories("./tests/common/c");
         CleanDirectories("./tests/common/mk");
-        CleanDirectories(GetDirectories("./tests/**/obj"));
-        CleanDirectories(GetDirectories("./tests/**/bin"));
+
+        DeleteDirectories(GetDirectories("./tests/**/obj"), new DeleteDirectorySettings {
+		Recursive = true,
```

Comment:
These look like they are indented with tabs and I think all cake files use spaces

Reply:
Woops. My bad. Fixing.